### PR TITLE
Update now 11.25 am.

### PR DESCRIPTION
**Pull request by thanyasol**

### CODE REVIEW

**Comment by thanyasol** [comment]
test now

**Comment by pikezeed** [comment]
ok confirm

File: Project-allday/application/models/product/main_product_model.php [hunk 1]
```diff
@@ -5,7 +5,7 @@ public function __construct()
 		  parent::__construct();
 		  $this->load->database();
     }	
-    /*### coding Name function should be getAll...Product ###*/ 
+    /*### coding Name function should be getAll...Product SOL ###*/
```
Comment (pikezeed) [inline Project-allday/application/models/product/main_product_model.php]: Good

**Comment by thanyasol** [comment]
I see it !!

**Comment by pikezeed** [comment]
again

**Comment by thanyasol** [comment]
OK now

**Pull request merged**